### ISSUE TITLE
OpenAPI doc is deprecated

### ISSUE THREAD

**Issue by lmazuel** [microsoft/cadl#104]
At the time of bug bash early Dec 2021:
https://github.com/microsoft/cadl/blob/main/docs/cadl-for-openapi-dev.md#path-item-object

```
@resource("/pets")
namespace Pets {
  op @get list(): Pet[];                  // get on path "/pets"
  op @get read(@path petId: int32): Pet;  // get on path "/pets/{petId}"
  @post("{petId}:walk")
  op walk(... PetId): {};                 // post on path "/pets/{petId}:walk"
}
```

is incorrect

**Comment by markcowl** [comment]
@lmazuel Fixed by this PR: #102 Please verify and close.

**Comment by lmazuel** [comment]
I think there is still a need to reverse the order of `op` and `@get` ? I start to forget my bush bash CADL day now :p
